README.md

```diff
@@ -96,6 +97,10 @@ Order go imports
 
 Run Go Revive linter
 
+### `generate-hook-docs`
+
+Generate markdown documentation from the hook descriptions in `.pre-commit-hooks.yaml` into the `README.md`. Docs are generated between `hooks-doc start` and `hooks-doc end` markdown comment blocks.
```

Comment:
I see that we're not doing that yet, but: should we start using the one sentence per line rule in this repo? (And hook it? 😁)